doc/language.md

```diff
@@ -801,6 +801,14 @@ this list, but most apps can safely use them:
   As a special exception to this rule, `List(Bool)` may **not** be upgraded to a list of structs,
   because implementing this for bit lists has proven unreasonably expensive.
 
+* A integer field may be promoted to an integer field of larger width while keeping the sign
+* A boolean field may be promoted to an integer field
+* A `Text` field may be promoted to `Data`
+* A `Data` field may be promoted to `List(UInt8)`
+* An enumeration may be promoted to `Uint16`, `UInt32`, or `UInt64`
```

Comment:
An enumeration can safely be changed to `UInt16`, because they are the same size. It cannot be changed to any larger type, for the same reason integers cannot be changed to larger types.